doofinder-for-woocommerce/includes/class-data-index.php

```diff
@@ -444,10 +444,12 @@ private function load_languages()
 
 		$languages = $this->language->get_languages();
 
+		$this->languages = [''];
```

Comment:
I would put this in the else, as it was before

Reply:
I think there's no need to add the else, as the default value is defined before and reinitialized in case $languages is an array.